beyond-essence/inventory/links.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Links
+background_image: index.jpg
+---
+
+Links are navigation elements.
+They are interactive and give users access to further content that is not directly accessible via the primary navigation.
+
+## Use cases
+
+In the Beyond cockpit, we provide various types of links:
+
+* Links in tables and overviews
+
+{% image_custom image="/assets/img/pages/essence/links-in-tables.png" width="50" lightbox %}
+
+* Links with icons
+
+{% image_custom image="/assets/img/pages/essence/links-with-icons.png" width="50" lightbox %}
+
+* Stand-alone links
+
+{% image_custom image="/assets/img/pages/essence/links-standalone.png" width="50" lightbox %}
+
+* Breadcrumb links
+
+{% image_custom image="/assets/img/pages/essence/links-breadcrumbs.png" width="50" lightbox %}
+
+* Links embedded in text
+
+{% image_custom image="/assets/img/pages/essence/links-embedded.png" width="50" lightbox %}
+
+* Pagination links
+
+{% image_custom image="/assets/img/pages/essence/links-pagination.png" width="50" lightbox %}
+
```

Comment:
Just for completeness, we also have only icons as links (Online Shop overview page):
<img width="180" alt="Bildschirmfoto 2019-08-09 um 14 33 55" src="https://user-images.githubusercontent.com/6104802/62779182-c2ffcd80-bab2-11e9-8f8d-50e490562c61.png">

Reply:
As already discussed: This is definitely a topic for later. If we sit together, let's discuss if we'd like to add these linked icons here or in the icons article.

Reply:
I took a note ;-)